components/dropdown/dropdown-menu.js

```diff
@@ -65,7 +65,13 @@ class DropdownMenu extends ThemeMixin(DropdownContentMixin(LitElement)) {
 		if (e.target.getAttribute('role') !== 'menuitemradio') {
 			return;
 		}
-		this.close();
+
+		// Add a small delay before closing the menu
+		const duration = 50;
```

Comment:
This can be declared as a global const at the top of the file (e.g., https://github.com/BrightspaceUI/core/blob/main/components/calendar/calendar.js#L15)